src/main/java/net/dries007/tfc/common/blocks/IClimateWeatheringBlock.java

```diff
@@ -0,0 +1,194 @@
+package net.dries007.tfc.common.blocks;
+
+import java.util.Iterator;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.core.particles.ParticleTypes;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.util.Mth;
+import net.minecraft.util.ParticleUtils;
+import net.minecraft.util.RandomSource;
+import net.minecraft.util.valueproviders.UniformInt;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.phys.BlockHitResult;
+
+import net.dries007.tfc.common.TFCTags;
+import net.dries007.tfc.common.items.TFCItems;
+import net.dries007.tfc.util.climate.Climate;
+
+public interface IClimateWeatheringBlock
+{
+    float[] AGE_MODIFIERS = {0.5f, 1.0f, 1.0f, 1.0f};
+
+    Block getNext();
```

Comment:
This should be using the Neo data map for weathering - I think doing it this way exposes us to other mods in a compatible way. Thus `getNext()` and `getPrevious()` are both unnecessary.